supervisor/bootstrap.py

```diff
@@ -216,6 +219,14 @@ def migrate_system_env(coresys: CoreSys) -> None:
         except OSError:
             _LOGGER.error("Can't cleanup old Add-on build directory at '%s'", old_build)
 
+    # Supervisor 2022.5 -> 2022.6. Can be removed after 2022.9
+    # pylint: disable=protected-access
```

Comment:
I don't like doing it this way but I realized I don't actually want APIs in StoreManager and CoreConfig for directly modifying their private data. This is a one time thing that should be removed in the future and I'd rather not create new supporting APIs around it.